pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -893,30 +893,34 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		domain.Spec.Devices.Rng = newRng
 	}
 
-	//usb controller is turned on, only when user specify input device with usb bus,
-	//otherwise it is turned off
-	if usbDeviceExists := isUSBDevicePresent(vmi); !usbDeviceExists {
-		// disable usb controller
-		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, Controller{
-			Type:  "usb",
-			Index: "0",
-			Model: "none",
-		})
-	}
-
+	isUSBDevicePresent := false
```

Comment:
hmm, there is a function with the same name on line 521
What am I missing here?

Reply:
ah, @ksimon1 can you please remove the function 
`func isUSBDevicePresent(vmi *v1.VirtualMachineInstance) bool {` on line 521
as @mfranczy suggested?